### PR TITLE
OSDOCS-12118 changed the node count reference in the service definition

### PR DESCRIPTION
**Pull request by xJustin**
Removed mentions of node count where it was redundant and linked users to the L&S doc.
Version(s):
4.17+

Issue:
https://issues.redhat.com/browse/OSDOCS-12118

Link to docs preview:

HCP Instance types:
https://82901--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-hcp-instance-types.html

Classic Instance Types:
https://82901--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-instance-types.html

ROSA HCP Service def:
https://82901--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-hcp-service-definition.html#rosa-sdpolicy-instance-types_rosa-hcp-service-definition

ROSA classic Service def:
https://82901--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-sdpolicy-instance-types_rosa-service-definition

Release note update:
https://82901--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_release_notes/rosa-release-notes#rosa-q3-2024_rosa-whats-new

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/rosa-sdpolicy-instance-types.adoc [hunk 1]
```diff
@@ -13,20 +13,13 @@ endif::[]
...
 
 ifdef::rosa-with-hcp[]
-All {hcp-title} clusters require a minimum of 2 worker nodes. All {hcp-title} clusters support a maximum of 500 worker nodes. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss.
-
-[NOTE]
-====
-Customers running {hcp-title} 4.14.x and 4.15.x clusters require a minimum z-stream version of 4.14.28 or 4.15.15 and greater to scale to 500 worker nodes. For earlier versions, the maximum is 90 worker nodes. 
-====
-
+All {hcp-title} clusters require a minimum of 2 worker nodes. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss.
```
Comment (bchandra-ocp) [inline modules/rosa-sdpolicy-instance-types.adoc:16]: What's the background to include the statement regarding shutting down is unsupported. It seems misplaced in this topic. We should remove that.

**Review by bchandra-ocp — CHANGES_REQUESTED** [review]
Requested to remove a line added regarding the shutting down of instances.

File: modules/rosa-sdpolicy-instance-types.adoc (hunk 1 above)
Comment (xJustin) [reply modules/rosa-sdpolicy-instance-types.adoc:16]: I will look into this.

**Review by radtriste — APPROVED** [review]
/lgtm

File: rosa_architecture/rosa_policy_service_definition/rosa-hcp-instance-types.adoc [hunk 2]
```diff
@@ -7,16 +7,13 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
...
-====
-
 include::modules/rosa-sdpolicy-am-aws-compute-types.adoc[leveloffset=+1]
 
 include::modules/rosa-sdpolicy-am-aws-compute-types-graviton.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
+For more information about the maximum number of supported worker nodes in a {hcp-title} cluster, see xref:../../rosa_planning/rosa-hcp-limits-scalability.adoc#rosa-hcp-limits-scalability[{hcp-title} limits and scalability].
```
Comment (jeana-redhat) [inline rosa_architecture/rosa_policy_service_definition/rosa-hcp-instance-types.adoc:17]: [Additional resources sections](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) should just be bullets with link text:
```suggestion
* xref:../../rosa_planning/rosa-hcp-limits-scalability.adoc#rosa-hcp-limits-scalability[{hcp-title} limits and scalability]
```

File: rosa_architecture/rosa_policy_service_definition/rosa-hcp-service-definition.adoc [hunk 3]
```diff
@@ -29,6 +29,7 @@ include::modules/rosa-sdpolicy-instance-types.adoc[leveloffset=+2]
 
 For a detailed listing of supported instance types, see xref:../rosa_policy_service_definition/rosa-hcp-instance-types.adoc#rosa-hcp-instance-types[{hcp-title} instance types].
 
+For more information about the maximum number of supported worker nodes in a {hcp-title} cluster, see xref:../../rosa_planning/rosa-hcp-limits-scalability.adoc#rosa-hcp-limits-scalability[{hcp-title} limits and scalability].
```
Comment (jeana-redhat) [inline rosa_architecture/rosa_policy_service_definition/rosa-hcp-service-definition.adoc:32]:
```suggestion
* xref:../rosa_policy_service_definition/rosa-hcp-instance-types.adoc#rosa-hcp-instance-types[{hcp-title} instance types]

* xref:../../rosa_planning/rosa-hcp-limits-scalability.adoc#rosa-hcp-limits-scalability[{hcp-title} limits and scalability]
```

File: rosa_architecture/rosa_policy_service_definition/rosa-instance-types.adoc [hunk 4]
```diff
@@ -13,4 +13,6 @@ include::modules/rosa-sdpolicy-am-aws-compute-types.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 
+For more information about the maximum number of supported worker nodes in a {rosa-classic} cluster, see xref:../../rosa_planning/rosa-limits-scalability.adoc#rosa-limits-scalability[Limits and scalability].
```
Comment (jeana-redhat) [inline rosa_architecture/rosa_policy_service_definition/rosa-instance-types.adoc:16]:
```suggestion
* xref:../../rosa_planning/rosa-limits-scalability.adoc#rosa-limits-scalability[Limits and scalability]
```

File: rosa_release_notes/rosa-release-notes.adoc [hunk 5]
```diff
@@ -18,7 +18,7 @@ toc::[]
 
 * **{hcp-title} multi-architecture cluster update.** {hcp-title-first} clusters created before 25 July, 2024 will migrate to a multi-architecture image on their next upgrade allowing you to use {AWS} Arm-based Graviton instance types for your workloads. For more information, see xref:../upgrading/rosa-hcp-upgrading.adoc#rosa-upgrade-options_rosa-hcp-upgrading[Upgrading ROSA with HCP clusters].
 
-* **{hcp-title} cluster node limit update.** {hcp-title} clusters can now scale to 500 worker nodes. This is an increase from the previous limit of 250 nodes. For more information, see xref:../rosa_planning/rosa-hcp-limits-scalability.adoc#tested-cluster-maximums-hcp-sd_rosa-hcp-limits-scalability[ROSA with HCP limits and scalability].
+* **{hcp-title} cluster node limit update.** {hcp-title} clusters can now scale to 500 worker nodes. This is an increase from the previous limit of 250 nodes. The 250 node limit was an increase from the previous limit 90 nodes on 26 August, 2024. For more information, see xref:../rosa_planning/rosa-hcp-limits-scalability.adoc#tested-cluster-maximums-hcp-sd_rosa-hcp-limits-scalability[ROSA with HCP limits and scalability].
```
Comment (jeana-redhat) [inline rosa_release_notes/rosa-release-notes.adoc:21]: The link actually goes to a different heading: ROSA with HCP cluster maximums
```suggestion
* **{hcp-title} cluster node limit update.** {hcp-title} clusters can now scale to 500 worker nodes. This is an increase from the previous limit of 250 nodes. The 250 node limit is an increase from the previous limit 90 nodes on 26 August, 2024. For more information, see xref:../rosa_planning/rosa-hcp-limits-scalability.adoc#tested-cluster-maximums-hcp-sd_rosa-hcp-limits-scalability[ROSA with HCP cluster maximums].
```

File: rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc [hunk 6]
```diff
@@ -29,6 +29,8 @@ include::modules/rosa-sdpolicy-instance-types.adoc[leveloffset=+2]
 
 For a detailed listing of supported instance types, see xref:../rosa_policy_service_definition/rosa-instance-types.adoc#rosa-instance-types[{product-title} instance types].
 
+For more information about the maximum number of supported worker nodes in a {rosa-classic} cluster, see xref:../../rosa_planning/rosa-limits-scalability.adoc#rosa-limits-scalability[Limits and scalability].
```
Comment (jeana-redhat) [inline rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc:32]:
```suggestion
* xref:../rosa_policy_service_definition/rosa-instance-types.adoc#rosa-instance-types[{product-title} instance types]

* xref:../../rosa_planning/rosa-limits-scalability.adoc#rosa-limits-scalability[Limits and scalability]
```

**Review by jeana-redhat — COMMENTED** [review]
It's all in existing content, but consider [s/worker/compute](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-conscious-language). Might not make sense to do this here, since you would want it to be consistent.

Most comments are just additional resources items out of compliance in the same way (including a couple existing that should probably be fixed for consistency).

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by xJustin** [comment]
Thanks @jeana-redhat ! I committed all of your suggestions

**Review by ShaunaDiaz — COMMENTED** [review]
/lgtm

**Comment by ShaunaDiaz** [comment]
@xJustin Looks like the `4.18` ROSA content is out of sync with the `main` branch. You'll want to investigate and troubleshoot that when you work on the manual cherry pick. The `main` and `4.18` branches should not be out of sync.

**Comment by bergerhoffer** [comment]
@xJustin @ShaunaDiaz I'm doing some commit checking and found another PR that hasn't been cherry picked properly to 4.18 yet. https://github.com/openshift/openshift-docs/pull/82624

We'll get that merged and then hopefully this will cherry pick cleanly afterward.

**Pull request merged**